hooks/useWriting.ts

```
@@ -1,8 +1,99 @@
+    y,
+    maxWidth,
+    lineHeight,
+  }: {
+    ctx: CanvasRenderingContext2D;
+    text: string;
+    x: number;
+    y: number;
+    maxWidth: number;
+    lineHeight: number;
+  }) => {
+    let words = text.replaceAll("\n", " \n ").split(/ +/);
```

ESLint

[error] 'words' is never reassigned. Use 'const' instead.

(prefer-const)